### PR TITLE
Use referer to allow origin and set x-frame-option

### PR DESCRIPTION
**Pull request by nchoi**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -27,11 +27,10 @@ marked.setOptions({
...
 });
 
-//Set x-frame-options to allow iframing from only our site
-app.use(frameguard({
-  action: 'allow-from',
-  domain: 'https://crisistextline.instructure.com/'
-}))
+const ALLOWED_BY = new Set([
+  'https://crisistextline.instructure.com/',
+  'https://home.crisistextline.org/'
```
Comment (jaybennett) [inline app.js]: We should check this, but I think this needs to be crisistextline.okta.com right @kidatsy ?

File: app.js [hunk 2]
```diff
@@ -59,8 +58,9 @@ app.get('/hello', function (req, res, next) {
 });
 
 app.get('/:id', function (req, res, next) {
-	const token = req.query.token;
-	if (token === app.get('secret')){
+	const referer = req.header('Referer');
+	if (ALLOWED_BY.has(referer)){
+    	res.setHeader('X-Frame-Options', 'ALLOW-FROM ' + domain);
```
Comment (jaybennett) [inline app.js]: Spacing?

Comment (nchoi) [inline app.js]: Fixed

Comment (jaybennett) [inline app.js]: Shouldn't line 63 be indented like 64?  That's what I meant.

Comment (nchoi) [inline app.js]: Yep, fixed it directly thru github.

https://github.com/CrisisTextLine/contentful-app/blob/master/app.js

**Pull request merged**